src/components/navMenu/navMenu.tsx

```diff
@@ -19,6 +19,7 @@ const NavMenu = ({ render }) => {
         },
         ":hover": {
           textDecoration: "none !important",
+          color: "ui.link.secondary",
```

Comment:
I forgot why this needs to be manually added. This style rule is already applied elsewhere 
<img width="315" alt="Screen Shot 2024-03-20 at 1 04 40 PM" src="https://github.com/NYPL/digital-collections/assets/1280564/3749e2b7-b18e-4f37-bfe6-0a3231820bd4">

Reply:
Ah it doesn't I thought I was overriding with the `color` and needed to reset